Directory.Packages.props

```diff
@@ -5,57 +5,57 @@
     <CentralPackageTransitivePinningEnabled>true</CentralPackageTransitivePinningEnabled>
   </PropertyGroup>
   <ItemGroup Label="Latest DotNet Package Versions. AutoUpdate" Condition="'$(TargetFramework)' == '$(LatestSupportedDotNetVersion)' OR '$(IsNetStandard)'">

```

Comment:
LatestSupportedDotNetVersion in Directory.Build.props will need to be updated to net9.0 too